pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -290,22 +290,24 @@ func NewContainerManager(mountUtil mount.Interface, cadvisorInterface cadvisor.I
 	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.TopologyManager) {
 		cm.topologyManager, err = topologymanager.NewManager(
 			numaNodeInfo,
-			nodeConfig.ExperimentalTopologyManagerPolicy,
+			nodeConfig.TopologyManagerPolicy,
 		)
 
 		if err != nil {
 			return nil, err
 		}
 
-		klog.Infof("[topologymanager] Initilizing Topology Manager with %s policy", nodeConfig.ExperimentalTopologyManagerPolicy)
+		klog.Infof("[topologymanager] Initializing Topology Manager with %s policy", nodeConfig.TopologyManagerPolicy)
 	} else {
-		cm.topologyManager = topologymanager.NewFakeManager()
+		cm.topologyManager = nil
```

Comment:
Are there any areas which we previously relied on the existence of a `FakeTopologyManager`, but now need to start checking the TopologyManager isn't nil?

Reply:
I thought this PR was OK, but now I see the device-plugin-gpu test keeps failing.. is it possible this is because of nil topologyManager being passed here? :/  
https://github.com/kubernetes/kubernetes/blob/1888745c3db2432a613b2649686ee6bf5c9d2fa1/pkg/kubelet/cm/devicemanager/manager.go#L121